proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -172,4 +166,46 @@ Since new parameters are added, Core, iOS, Java Suite, RPC, Protocol, and HMI ar
 
 ## Alternatives considered
 
-Nothing.  
+The flow shown in Proposed solution is complicated, so the conditions have been arranged.
```

Comment:
I'm having trouble understanding the meaning of this sentence, specifically "so the conditions have been arranged."  Can you please clarify and re-word?

You could update to "The flow in the `Proposed solution` section is complicated, so the author has also considered the simplified solution below." if this is your intended meaning.

Reply:
Sorry for the poor English translation. What I intended is as you wrote.